Formula/gofabric8.rb

```diff
@@ -0,0 +1,22 @@
+class Gofabric8 < Formula
+  desc "CLI for fabric8 running on Kubernetes or OpenShift"
+  homepage "https://github.com/fabric8io/gofabric8/"
+  url "https://github.com/fabric8io/gofabric8/archive/v0.4.71.tar.gz"
+  sha256 "4eacee14e73056933dab263918cc64e45c57a2c7c7575fd23dada2a6ff3c5fe1"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    files = Dir["*"]
+    mkdir buildpath/"src/github.com/fabric8io/gofabric8" do
+      files.each { |f| mv buildpath/f, "." }
```

Comment:
I changed it to the current approach based upon previous feedback by Mike.

I'm perfectly happy to take the approach you suggest, as long as this seems like a solution we can stick with. 

It does seem substantially more compact, so I have no complaints and will change it unless someone suggests a motivation for not doing so (e.g. clarity or otherwise).